src/features/hotspots/details/HotspotDetailCardHeader.tsx

```diff
@@ -20,7 +20,7 @@ const HotspotDetailCardHeader = ({
         <Heart />
       </TouchableOpacityBox>
       <CardHandle />
-      <TouchableOpacityBox onPress={onMoreSelected}>
+      <TouchableOpacityBox onPress={onMoreSelected} padding="m" margin="n_m">
```

Comment:
just curious what the padding + negative margin does here?

Reply:
It gives some hit slop. The button needed some padding, but the container had the padding already. This gives it the padding, then the n_m repositions it where it should be.